test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -92,6 +91,51 @@ static public void testCreatePoint2D() throws Throwable {
 		assertEquals(getY.invoke(point2D), yNew);
 	}
 
+	/*
+	 * Create a safe value type with multibyte primative members
+	 * 
+	 * value Point2DSafe {
+	 * 	double x;
+	 * 	long y;
+	 * }
+	 */
+	@Test(priority=1)
+	static public void testCreatePoint2DComplex() throws Throwable {
+		String fields[] = {"x:D", "y:J"};
+		Class point2DClassComplex = ValueTypeGenerator.generateValueClass("Point2DComplex", fields);
+		
+		MethodHandle makePoint2DSafeComplex = lookup.findStatic(point2DClassComplex, "makeValue", MethodType.methodType(point2DClassComplex, double.class, long.class));
+
+		MethodHandle getXComplex = generateGetter(point2DClassComplex, "x", double.class);
```

Comment:
perhaps change the field names of x and y to something else. This way getter names cannot be confused

Reply:
Changed to d and j to match the data types used